pkg/utils/nodeclass/nodeclass.go

```diff
@@ -72,7 +72,7 @@ func NewSubnetSelectorTerms(subnetSelector map[string]string) (terms []v1beta1.S
 	for k, v := range subnetSelector {
 		switch k {
 		case "aws-ids", "aws::ids":
-			ids = strings.Split(strings.Trim(v, " "), ",")
+			ids = lo.Map(strings.Split(v, ","), func(s string, _ int) string { return strings.Trim(s, " ") })
 		default:
 			tags[k] = v
```

Comment:
Based on the changes that you've made, we now need to split the tag value here if it contains a comma